From 2d244a80340fe4ad66218f8cca1fe84119172ce3 Mon Sep 17 00:00:00 2001
Message-Id: <2d244a80340fe4ad66218f8cca1fe84119172ce3.1335361915.git.minovotn@redhat.com>
In-Reply-To: <ce1a7d1539a0b4b36555d1035257f57af7ae8478.1335361915.git.minovotn@redhat.com>
References: <ce1a7d1539a0b4b36555d1035257f57af7ae8478.1335361915.git.minovotn@redhat.com>
From: Paolo Bonzini <pbonzini@redhat.com>
Date: Tue, 24 Apr 2012 14:01:35 +0200
Subject: [PATCH 8/8] mirror: do not reset sector_num

RH-Author: Paolo Bonzini <pbonzini@redhat.com>
Message-id: <1335276095-25813-9-git-send-email-pbonzini@redhat.com>
Patchwork-id: 39433
O-Subject: [RHEL 6.3 qemu-kvm PATCH 8/8] mirror: do not reset sector_num
Bugzilla: 813862
RH-Acked-by: Marcelo Tosatti <mtosatti@redhat.com>
RH-Acked-by: Kevin Wolf <kwolf@redhat.com>
RH-Acked-by: Jeffrey Cody <jcody@redhat.com>

Bugzilla: 813862

RHEL-only.
---
 block/mirror.c |    1 -
 1 file changed, 1 deletion(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 block/mirror.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/block/mirror.c b/block/mirror.c
index 03e4db8..1460031 100644
--- a/block/mirror.c
+++ b/block/mirror.c
@@ -166,7 +166,6 @@ static void coroutine_fn mirror_run(void *opaque)
              * used to pivot to the mirroring target.
              */
             synced = true;
-            sector_num = -1;
             s->common.offset = end * BDRV_SECTOR_SIZE;
         }
 
-- 
1.7.7.6

